src/main/java/seedu/bigpp/command/viewercommand/ViewerDeleteCommand.java

```diff
@@ -0,0 +1,29 @@
+package seedu.bigpp.command.viewercommand;
+
+import seedu.bigpp.command.Command;
+import seedu.bigpp.pc.PC;
+import seedu.bigpp.pc.PCList;
+
+public class ViewerDeleteCommand extends Command {
+
+    public ViewerDeleteCommand(String arguments) {
+        super.setArguments(arguments);
+    }
+
+    /**
+     * Delete a PC of a given index from the PC list
```

Comment:
Good practice to add documentation to new methods

Reply:
Will use this format for future documentation